corpora/csip/metadata/amdsec/CSIP31/test-case.xml

```diff
@@ -11,14 +11,85 @@ XML lists -->
   <!-- The full text of the requirement. -->
   <requirementText>If administrative / preservation metadata is available, it must be described using the administrative metadata section (amdSec) element.
 It is possible to transfer metadata in a package using just the descriptive metadata sectiond/or adminstrative metadata section.
+Cardinality and Level: 0..n SHOULD
 </requirementText>
   <!-- Textual description of the test case, extra notes beyond the requirment text. -->
-  <description></description>
+  <description>
+    This testcase is for requirement CSIP31, which relates to administrative metadata; the optional amdSec elements in the mets.xml file and to the optional files in the folder metadata/descriptive.
+    The requirement CSIP31 is not stringent and therefore difficult to test, it is based on conditions and optional elements.
+    The requirement text about 'It is possible to transfer metadata in a package using just the descriptive metadata sectiond/or adminstrative metadata section.' does not give much meaning here.
+    
+      </description>
   <!-- List of requirments that this test case depends on in addition to the
   main requirememt, e.g. general requirments on the form of IDs -->
   <dependencies>
+    <dependency requirementId="CSIPSTR6" URL="http://earkcsip.dilcis.eu/#41-folder-structure-of-the-csip"></dependency>
   </dependencies>
   <!-- A list of the validation rules derived from the test case -->
   <rules>
+    
+    <rule id="1">
+      <!-- Text description of the validation rule -->
+      <description>There should be an XML element named amdSec in the mets.xml</description>
+      <!-- Details of the validation error generated by the message, can be one of: [ ERROR | WARNING | INFO ]-->
+      <error level="WARNING">
+        <!-- Expected error message when the rule is violated. -->
+        <message>The element mets/amdSec does not exist in METS.xml. It SHOULD be there.</message>
+      </error>
+      <corpusPackages>
+        <!-- Details of corpus packages designed to test this validation rule. -->
+        <package name="IP_18000_CSIP31_1" isValid="FALSE">
```

Comment:
I am not sure what the purpose of isValid is? Should it be false if the package should produce a warning or an error? and true if it should produce no warning or error?